VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -227,14 +227,34 @@ def get_http_util(self):
         from HttpUtil import HttpUtil
         return HttpUtil(self.logger)
 
+    def get_wireserver_endpoint_uri(self):
+
+        wireserver_endpoint_file = CommonVariables.wireserver_endpoint_file
+        wireserver_IP = None
+        wireserver_endpoint_uri = CommonVariables.wireserver_endpoint_uri
+
+        if os.path.exists(wireserver_endpoint_file):
+            with open(wireserver_endpoint_file, 'r') as wip:
+                wireserver_IP = wip.read()
+                self.logger.log("wireserver_IP found in {0} = {1}".format(wireserver_endpoint_file, wireserver_IP))
+
+        if wireserver_IP == None:
```

Comment:
style: use "is None" instead of "== None"

Reply:
FWIW https://www.python.org/dev/peps/pep-0008/